sympy/codegen/ast.py

```diff
@@ -151,7 +193,7 @@ def aug_assign(lhs, op, rhs):
         subclass these types are also supported.
 
     op : str
-        Operator (+, -, /, *, %).
+        Operator (+, -, /, \*, %).
```

Comment:
It's a Sphinx thing (asterisk needs to be escaped or it will be a reference), `\*` is interpreted as `\\*` by both Python 2 and Python 3. I will change this to '\\*' though.